TW-1102 Postbot section (#4922)
postmanlabs/postman-docs: src/pages/docs/writing-scripts/test-scripts.md

* Check if there are any errors in your scripts. A red badge will highlight scripts with errors. You can also check the response section for specific errors.
* Debug your tests using the [log statements](/docs/sending-requests/troubleshooting-api-requests/#using-log-statements) to ensure that you are asserting on correct data.

## Using Postbot to write tests

> Not sure how to write an test script for your request? Ask Postbot! You can use artificial intelligence to write your test scripts. You can ask Postbot a simple text question, and it will generate a test to solve your question. You can use Postbot to add a new set of tests, visualize responses, save a field from a response, fix your existing tests, and more.

Reviewer: ```suggestion
Write a sentence that describes the goal of your test in Postbot, and it provides several prompts to generate a test. You can use Postbot to add a new set of tests, visualize responses, save a field from a response, and fix your existing tests.
```

To write a test with Postbot, do the following:

1. Send your request so it has a response.
1. Go to the **Tests** tab.
1. In the **Script with Postbot** section next to your test, select the Postbot icon <img alt="Postbot icon" src="https://assets.postman.com/postman-docs/v10/icon-postbot-v10-16.jpg#icon" width="18px">.

> 1. Enter enter your query as a simple text sentence, or select a built-in suggestions to quickly tell Postbot what you need.

Reviewer 2: ```suggestion
1. Enter your query as a simple text sentence, or select one of the suggested queries to tell Postbot what you need.
```

> 1. Enter enter your query as a simple text sentence, or select a built-in suggestions to quickly tell Postbot what you need.

Reviewer: ```suggestion
1. Enter your query as a simple sentence, or select a built-in prompt to tell Postbot what you need.
```

      <img alt="Postbot writing a test" src="https://assets.postman.com/postman-docs/v10/postbot-add-test-v10-16.gif" width="300px" />

> 1. Select the play button. Postbot will write a test for you.

Reviewer: ```suggestion
1. Select the play button. Postbot writes a test for you.
```
